web/src/js/authentication/__tests__/user.test.js

```diff
@@ -580,3 +582,27 @@ describe('updateUserEmail tests', () => {
     })
   })
 })
+
+describe('deleteUser tests', () => {
+  it('works as expected', async () => {
+    const mockDeleteUser = jest.fn(() => Promise.resolve())
+
+    // A user must exist to be able to send a verification email.
+    const __setFirebaseUser = require('firebase/app').__setFirebaseUser
+    __setFirebaseUser({
+      uid: 'xyz987',
+      email: 'foo@example.com',
+      isAnonymous: false,
+      emailVerified: true,
+      getIdToken: jest.fn(() => 'fake-token-123'),
+      delete: mockDeleteUser,
+    })
+
+    const deleteUser = require('js/authentication/user').deleteUser
+    const response = await deleteUser()
+
+    // expect(DeleteUserMutation).toHaveBeenCalled()
```

Comment:
Why is this not working?

Reply:
When calling `jest.resetModules()` between tests, as we do here in the `afterEach` test teardowns, ES6 imports won't work. Instead, you have to require modules within tests. Add this:
`const DeleteUserMutation = require('js/mutations/DeleteUserMutation').default`

Reply:
Ah! Thanks